src/components/Navigation.tsx

```diff
@@ -8,20 +8,23 @@ import { NAVIGATION } from '../constants';
 import type { Tabs } from '../constants';
 
 export function Navigation() {
-  const [active, setActive] = useState<Tabs>('portal');
+  const [active, setActive] = useState<Tabs>('home');
   const router = useRouter();
 
   useEffect(() => {
-    let category = 'portal';
-    if (router.pathname !== '/') {
+    let category = 'home';
```

Comment:
some question is: why we are using the `category` prop in the frontmatter if we are setting this manually?